.github/workflows/dockerhub-release.yml

```diff
@@ -72,7 +72,7 @@ jobs:
           IMAGES=$(cat .github/promote-images.yml | yq '."europe-docker.pkg.dev/gitpod-artifacts/docker-dev"."images-by-tag-regex"|keys[]' -r)
           for IMAGE in $IMAGES;
           do
-            sudo skopeo copy \
+            sudo skopeo copy --format=oci --dest-oci-accept-uncompressed-layers \
```

Comment:
I have a question. Why is it worth uncompressing?

Reply:
for research?

Reply:
> Why is it worth uncompressing?

We are just copying something. If we want to compress it, let's do that during the build, not the copy.
At this point, this should be dumb.

Reply:
> for research?

This is the first step to anything interesting in the future, like [SOCI](https://github.com/awslabs/soci-snapshotter)

Reply:
I see! Thanks for your explanation :pray: 
By the way, can we think of using `zstd:chunked`?

Reply:
> By the way, can we think of using zstd:chunked?

Yes? But I think this is missing in buildkit https://github.com/opencontainers/image-spec/issues/803